packages/fxa-settings/src/components/en.ftl

```diff
@@ -20,10 +20,14 @@
 # “Account” can be localized, “Firefox” must be treated as a brand.
 # This is used to refer to a user's account, e.g. "update your Firefox account ..."
 -product-firefox-account = Firefox account
-product-mozilla-vpn = Mozilla VPN
-product-pocket = Pocket
-product-firefox-monitor = Firefox Monitor
-product-firefox-relay = Firefox Relay
+-product-mozilla-vpn = Mozilla VPN
+-product-mozilla-mdn = Mozilla MDN
```

Comment:
nit: the "M" in MDN is for Mozilla. Making this "_Mozilla_ Mozilla Developer Network" (which would probably be the same as saying "ATM Machine").
Not sure if the answer is saying "MDN" without the "Mozilla" prefix (if we think enough general people would understand the acronym) or possibly "Mozilla Developer Network".

We might need to ping the MDN folks and ask them how they want to be represented on the list.

Reply:
```suggestion
-product-mdn-plus = MDN Plus
```
Per Vesta from [JIRA ticket](https://mozilla-hub.atlassian.net/browse/FXA-6374)